examples/models/file/glyphs.py

```diff
@@ -1,15 +1,15 @@
 import numpy as np
 
-from bokeh.models import ColumnDataSource, Plot, LinearAxis, Grid, HoverTool
-from bokeh.models.widgets import Paragraph
-from bokeh.models.layouts import Column, Tabs, Panel
-from bokeh.models.glyphs import (
-    AnnularWedge, Annulus, Arc, Bezier, Circle, Dash, ImageURL, Line, MultiLine, MultiPolygons, Oval, Hex,
-    Patch, Patches, Quad, Quadratic, Ray, Rect, Segment, Square, Text, Wedge, CircleX, Triangle,
-    Cross, Diamond, InvertedTriangle, SquareX, Asterisk, SquareCross, DiamondCross, CircleCross, X
-)
 from bokeh.document import Document
 from bokeh.embed import file_html
+from bokeh.models import (AnnularWedge, Annulus, Arc, Asterisk, Bezier, Circle,
+                          CircleCross, CircleX, Column, ColumnDataSource,
+                          Cross, Dash, Diamond, DiamondCross, Grid, Hex,
+                          HoverTool, ImageURL, InvertedTriangle, Line,
+                          LinearAxis, MultiLine, MultiPolygons, Oval, Panel,
+                          Paragraph, Patch, Patches, Plot, Quad, Quadratic,
+                          Ray, Rect, Segment, Square, SquareCross, SquareX,
+                          Tabs, Text, Triangle, Wedge, X)
```

Comment:
What was wrong with the original indentation? It's much easier to work with short indentation that with this. It also wastes a lot less vertical space. The only reason this is even a remotely feasible option, is because `bokeh.models` is a fairly short module name. As far I can tell, the standard in Python is the former. PEP 328, which introduced multi-line imports, suggests this as well.

Reply:
it's the default for `isort` which is what I use to reformat most of these. I'd like to eventually integrate `isort` in to our CI pipeline. I'd say:
* personally I find this style (which is quite common) easier on the eyes
* import lists this long are outliers and should not weigh heavily on the style choice
* we can configure the `isort` line length to be longer which would also help in these cases
* consistency is the eventual goal

In any case `isort` is heavily configurable, so let's defer these decisions to a separate issue/PR that integrates it into our CI system.